### DISCUSSION TITLE
Events not emitted when multiple actions used

### DISCUSSION THREAD

**Discussion by borkowsky** [feathersjs/feathers#2279] [Q&A]
Server not send service events when i use multiple patch.
Channels ok:
`app.service('messages').publish(data => {
  return app.channel('authenticated')
})`
My request from frontend:
`$client.service('messages').patch(null, {
  readed: true
}, {
  query: {
    readed: false
  }
}).catch(e => {...do smth})`
Result in after.patch hook is empty.

**Comment by daffl** [comment]
Please add the information requested in the issue template if possible. Specifically, which database and database adapter module, what Node version, error message, what the hook is supposed to do etc. All I can say for now is that the channel publisher does not have anything to do with the `result` in an `after` hook.

**Comment by borkowsky** [comment]
> Please add the information requested in the issue template if possible. Specifically, which database and database adapter module, what Node version, error message, what the hook is supposed to do etc. All I can say for now is that the channel publisher does not have anything to do with the `result` in an `after` hook.

Database: Postgresql,
Database adapter: Feathers-sequelize 6.1.0
Node.js version 12.6.1,
No errors was thrown

**Comment by borkowsky** [comment]
We have any solutions of this problem?
